mllib/src/main/scala/org/apache/spark/mllib/clustering/LDAOptimizer.scala

```diff
@@ -573,7 +584,8 @@ private[clustering] object OnlineLDAOptimizer {
       expElogbeta: BDM[Double],
       alpha: breeze.linalg.Vector[Double],
       gammaShape: Double,
-      k: Int): (BDV[Double], BDM[Double], List[Int]) = {
+      k: Int,
+      epsilon: Double = 1e-3): (BDV[Double], BDM[Double], List[Int]) = {
```

Comment:
This duplicates the default value.

Reply:
This is for some other code,  like in LDAModel, which calls this function and does not need to set epsilon.

Reply:
shouldn't epsilon be set in LDAModel as well to reflect the value which was used while training the model?

Reply:
Maybe not need to set, we can add epsilon as a parameter in the model to save it. Because most of training parameters are not in the model, so we don't add epsilon to the model here. Thanks.